konnectivity-client/pkg/client/client.go

```diff
@@ -125,8 +125,11 @@ func (t *grpcTunnel) serve(c clientConn) {
 				select {
 				case ch <- result:
 				default:
-					klog.ErrorS(fmt.Errorf("blocked pending channel"), "Received second dial response for connection request", "connectionID", resp.ConnectID, "dialID", resp.Random)
-					// On multiple dial responses, avoid leaking serve goroutine.
+					// If there are no readers of the pending dial channel above, it means one of two things:
```

Comment:
I don't think we want to lose the logging for exiting dial_rsp here. Happy to talk about what the right log level is but we should be logging this.

Reply:
Added a V(1) log, similar to the one on line 118